### PR TITLE
Update the CDH version 5.7.0. 

### PR DESCRIPTION
**Pull request by awholegunch**
And consolidate the change into one place, while we are at it.

Passes [Quick Build 2](http://builds.cask.co/browse/CDAP-DOB179-2)

[Hadoop Compatibility](http://builds.cask.co/artifact/CDAP-DOB179/shared/build-2/Docs-HTML/3.4.0-SNAPSHOT/en/admin-manual/hadoop-compatibility.html)

### CODE REVIEW

File: cdap-docs/admin-manual/source/hadoop-compatibility.rst [hunk 1]
```diff
@@ -47,42 +47,47 @@ CDAP currently requires Internet access on the CDAP service nodes (or until the
 
 For a Distributed CDAP cluster, version |release|, you **must** install these Hadoop components:
 
+.. |cdh-version|       replace:: 5.0.0 through 5.7.0
+.. |cdh-version-hbase| replace:: 5.0.0 through 5.7.0 (1.0-cdh5.7.0)
```
Comment (wolf31o2) [inline cdap-docs/admin-manual/source/hadoop-compatibility.rst]: 1.2-cdh5.7.0

File: cdap-docs/admin-manual/source/hadoop-compatibility.rst [hunk 2]
```diff
@@ -47,42 +47,47 @@ CDAP currently requires Internet access on the CDAP service nodes (or until the
...
+.. |hdp-version|       replace:: 2.0 through 2.3
+.. |mapr-version|      replace:: 4.1 and 5.0
```
Comment (wolf31o2) [inline cdap-docs/admin-manual/source/hadoop-compatibility.rst]: 5.1

File: cdap-docs/admin-manual/source/hadoop-compatibility.rst [hunk 3]
```diff
@@ -93,19 +98,19 @@ Hadoop components, as required:
...
-|                | Cloudera Data Hub (CDH)         | 5.0.0 through 5.6.0                   |
+|                | Cloudera Data Hub (CDH)         | |cdh-version|                         |
 +                +---------------------------------+---------------------------------------+
-|                | Hortonworks Data Platform (HDP) | 2.0 through 2.3                       |
+|                | Hortonworks Data Platform (HDP) | |hdp-version|                         |
 +                +---------------------------------+---------------------------------------+
-|                | MapR                            | 4.1 and 5.0                           |
+|                | MapR                            | |mapr-version|                        |
 +----------------+---------------------------------+---------------------------------------+
 | **Spark**      | Apache                          | Versions 1.2.x, 1.3.x, and 1.4.x      |
```
Comment (wolf31o2) [inline cdap-docs/admin-manual/source/hadoop-compatibility.rst]: Versions 1.2.x through 1.6.x

**Comment by wolf31o2** [comment]
Couple comments to be resolved... Otherwise, 👍 LGTM

File: cdap-docs/admin-manual/source/hadoop-compatibility.rst [hunk 4]
```diff
@@ -47,42 +47,47 @@ CDAP currently requires Internet access on the CDAP service nodes (or until the
 
 For a Distributed CDAP cluster, version |release|, you **must** install these Hadoop components:
 
+.. |cdh-version|       replace:: 5.0.0 through 5.7.0
+.. |cdh-version-hbase| replace:: 5.0.0 through 5.7.0 (1.0-cdh5.7.0)
+.. |hdp-version|       replace:: 2.0 through 2.3
```
Comment (dereklwood) [inline cdap-docs/admin-manual/source/hadoop-compatibility.rst]: hdp: 2.0 through 2.4

File: cdap-docs/admin-manual/source/hadoop-compatibility.rst (hunk 1 above)
Comment (awholegunch) [inline cdap-docs/admin-manual/source/hadoop-compatibility.rst]: Fixed.

File: cdap-docs/admin-manual/source/hadoop-compatibility.rst (hunk 3 above)
Comment (awholegunch) [inline cdap-docs/admin-manual/source/hadoop-compatibility.rst]: Fixed.

File: cdap-docs/admin-manual/source/hadoop-compatibility.rst (hunk 4 above)
Comment (awholegunch) [inline cdap-docs/admin-manual/source/hadoop-compatibility.rst]: Fixed.

File: cdap-docs/admin-manual/source/hadoop-compatibility.rst (hunk 2 above)
Comment (awholegunch) [inline cdap-docs/admin-manual/source/hadoop-compatibility.rst]: Should this still include 5.0? In other words, be:
```
4.1 through 5.1

Comment (wolf31o2) [inline cdap-docs/admin-manual/source/hadoop-compatibility.rst]: Correct.

Comment (awholegunch) [inline cdap-docs/admin-manual/source/hadoop-compatibility.rst]: Fixed.

**Pull request merged**